src/cmd/deploy.go

```diff
@@ -4,57 +4,45 @@ import (
 	"fmt"
 	"log"
 	"os"
-	"path/filepath"
 
 	"github.com/Originate/exosphere/src/application"
-	"github.com/Originate/exosphere/src/config"
-	"github.com/Originate/exosphere/src/types"
-	"github.com/Originate/exosphere/src/util"
 	"github.com/spf13/cobra"
 )
 
-//deployCmd represents the deploy command
-var deployCmd = &cobra.Command{
-	Use:   "deploy",
-	Short: "Deploys Exosphere application to the cloud",
-	Long:  "Deploys Exosphere application to the cloud",
+var deployInit = &cobra.Command{
```

Comment:
We've been keeping subcommands in the same file as the root command (see `cmd/configure.go`). This is a subcommand of `exo deploy` invoked by `exo deploy init`.